main/medium/src/reaper.rs

```diff
@@ -6972,6 +7020,16 @@ pub struct TimeMapGetMeasureInfoResult {
     pub tempo: Bpm,
 }
 
+#[derive(Copy, Clone, PartialEq, Debug)]
+pub struct TimeMapQNToMeasuresResult {
```

Comment:
I follow Rust API conventions when it comes to acronyms (https://rust-lang.github.io/api-guidelines/naming.html).

```suggestion
pub struct TimeMapQnToMeasuresResult {
```